fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/PushDownPredicateScanRule.java

```diff
@@ -52,6 +52,7 @@ public List<OptExpression> transform(OptExpression input, OptimizerContext conte
         Preconditions.checkState(predicates != null);
         predicates = scalarOperatorRewriter.rewrite(predicates,
                 ScalarOperatorRewriter.DEFAULT_REWRITE_SCAN_PREDICATE_RULES);
+        predicates = Utils.transTrue2Null(predicates);
```

Comment:
Why need true to null?

Reply:
predicate is true means don't need predicate